xmodule/static_content.py

```diff
@@ -297,9 +297,10 @@ def main():
         installed_apps += ('edxval',)
     except ImportError:
         pass
-    settings.configure(
-        INSTALLED_APPS=installed_apps,
-    )
+    if not settings.configured:
```

Comment:
with new Dockerfile, settings are already configured prior to this step, which is causing an error to be thrown. Added a check to ensure that settings are not already configured before trying to configure them. Similar logic can be found here: https://github.com/openedx/edx-platform/blob/c2e6d96ea8dee90bda4d26014f9f1e74c7ee4f4a/xmodule/modulestore/django.py#L19